### DISCUSSION TITLE
Find eForms before being published (production)

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#535] [Q&A] [answered]
Where can the eForms sent by a eSender in production be found before they have been published?

In [https://enotices2.ted.europa.eu/consult-notices](url)?

![image](https://github.com/OP-TED/eForms-SDK/assets/115476802/c32139cc-0a9c-4876-aaa2-d40d3db9f7cf)

Thanks in advance,
Alex

**Comment by ferrakaop** [comment, marked as answer]
This is not a technical question and doesn't concern the SDK. It should be addressed to the TED helpdesk.

In general we don't recommend using the eNotices2 web front-end if you submit notices via API. You can query the notice status via API. But you could view "your" notices if the EU Login on eNotices2 is the same used to submit via API, even if this function is not guaranteed.

**Comment by AnnaKSZ** [comment]
Hi Alex. 

Yes, we can see our API eforms on this site.

> In [https://enotices2.ted.europa.eu/consult-notices](url)?
This is the productioon system's link.

But this is the test system's link.
https://enotices2.preview.ted.europa.eu/consult-notices

Regards
Anna
